lib/kwtool/accounts/schema/user_schema.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Kwtool.Accounts.Schema.UserSchema do
```

Comment:
In the soon-to-be-updated conventions, the `schema` suffix is removed: https://github.com/nimblehq/compass/issues/511. Let's start following these conventions.

Reply:
Updated.